src/StatCache.cc

```diff
@@ -59,7 +59,7 @@ int StatCache::stat(const char *path, struct stat *st)
 		}
 		statMap.erase(iter);                            // Remove stale entry
 	}
-	CacheEntry entry{};                               // Make a new entry
+	CacheEntry entry;                                 // Make a new entry
```

Comment:
During a compilation you get a warning for every member that isn't explicitly initialised. Daft because it is guaranteed to be zero.

The structure doesn't need initialised because if stat fails it isn't used, so I removed the initialisers to get rid of the warnings.